gradoop-common/src/main/java/org/gradoop/common/model/impl/pojo/Edge.java

```diff
@@ -101,4 +103,18 @@ public String toString() {
     return String.format("(%s)-[%s]->(%s)",
       sourceId, super.toString(), targetId);
   }
+
+  /**
+   * Returns a GDL formatted edge string.
+   * @param idToVertexName Maps GradoopId of a vertex to a string that represents the gdl
+   *                       variable name
+   * @return A GDL formatted edge string.
+   */
+  public String toGDLString(Map<GradoopId, String> idToVertexName) {
```

Comment:
Please remove this method from Edge. As @p-f mentioned we should keep core classes small.
Furthermore I dislike the 'wall of capitals' pattern. IMHO it should be `toGdlString`  even though we don't have a convention for this right now.